docs/becoming-a-maintainer/translations/es/como-configurar-tu-proyecto.md

```diff
@@ -0,0 +1,208 @@
+>
+> Los mantenedores del proyecto que no sigan o apliquen el Código de conducta de buena fe pueden enfrentar repercusiones temporales o permanentes según lo determine el resto del liderazgo del proyecto.
+
+### Pacto de colaboradores
+
+El Pacto de colaboradores es un código de conducta que puedes usar y adaptar para tus proyectos Open Source. Para obtener más información, visita el [sitio web oficial](https://www.contributor-covenant.org/).
+
+## Pautas de contribución
+
+Un archivo CONTRIBUTING contiene pautas sobre cómo los colaboradores pueden ayudar con tu proyecto. Generalmente se encuentra en el directorio raíz del proyecto para que todos puedan acceder a él fácilmente.
+
+Tu guía debe incluir las siguientes secciones:
+
+- Instrucciones de configuración del proyecto
+- Pautas para crear nuevos problemas como informes de errores y solicitudes de características
+- Información sobre qué problemas están disponibles para la contribución
+- Pautas para convenciones de commit para solicitudes de extracción
+- Pautas para convenciones de código y estilo
+- Pautas para crear solicitudes de extracción
+
+Para una mirada en profundidad a la creación de un archivo CONTRIBUTING sólido, lee [este artículo](https://dev.to/opensauced/how-to-make-a-delicious-contributing-guide-4bp3) de OpenSauced.
+
+## Plantillas de problema
+
+Las plantillas de problema ayudan a guiar a los colaboradores a proporcionar la información específica y estructurada necesaria al abrir problemas del proyecto. Tener estas plantillas en tu repositorio garantizará que recibas la información deseada para priorizar el problema correctamente.
+
+Puedes crear varias plantillas de problema, como informes de errores, solicitudes de características, actualizaciones de documentación, etc. Dentro de estas plantillas, puedes tener campos obligatorios como los pasos para reproducir el error o una sección de detalles para una solicitud de característica. También puedes adjuntar etiquetas específicas como `feature`, `needs triage`, o `bug` para informar los tipos de plantillas de problema.
+
+### Creación de plantillas de problema
+
+Hay dos formas de crear plantillas de problema en GitHub.
+
+#### 1. Usando el generador de plantillas de GitHub
+
+Para crear plantillas de problema usando el generador de plantillas de GitHub, deberás:
+
+- Ir a la configuración de tu proyecto,
+- Navegar a la sección "Características",
+- Haz clic en "Configurar plantillas" debajo de "Problemas".
```

Comment:
Similar to above, this is an instruction for navigating GitHub which has these buttons in english. 

- Haz clic en "Set up templates" debajo de "Issues".